islandora.module

```diff
@@ -1685,8 +1690,12 @@ function islandora_object_access($op, $object, $user = NULL) {
  */
 function islandora_islandora_object_access($op, $object, $user) {
   module_load_include('inc', 'islandora', 'includes/utilities');
-
-  return islandora_namespace_accessible($object->id) && user_access($op, $user);
+  if ($object->state != 'A') {
+    return islandora_namespace_accessible($object->id) && user_access($op, $user) && user_access(ISLANDORA_ACCESS_INACTIVE_AND_DELETED_OBJECTS, $user);
```

Comment:
So the question is how we ensure people that want A) can live in a world where B) and C) is possible and also ensure, documentation is clear, updates run and toggle (love forms) are available to do so. Sounds reasonable. 

I have mixed feelings on "preserve existing functionality and user experience" if the user experience is already confusing or limited. But seems like giving people access of current (even when bad) user experience and the choice to a better is good for now. Then this can open the discussion about 2x parallel ways versus deprecating in time first way in favor of a better implementation. Deprecation can be as slow as needed of course if any at all.